<!DOCTYPE html>
<html>

<head>
    <meta charset="UTF-8">
    <link rel="stylesheet"
        href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/markdown.css">
    <link rel="stylesheet"
        href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/highlight.css">
    <style>
        body {
            font-family: -apple-system, BlinkMacSystemFont, 'Segoe WPC', 'Segoe UI', system-ui, 'Ubuntu', 'Droid Sans', sans-serif;
            font-size: 14px;
            line-height: 1.6;
        }

        a {
            color: rgb(0, 97, 177)
        }
    </style>
    <style>
        .task-list-item {
            list-style-type: none;
        }

        .task-list-item-checkbox {
            margin-left: -20px;
            vertical-align: middle;
            pointer-events: none;
        }
    </style>

</head>

<body class="vscode-body vscode-light">

    <div>
        <div style="float:left; width: 75%;">
            <strong>About me</strong>
            <p>My name is Yoann Pageaud.<br>
                I am a PhD student in cancer bioinformatics.<br>
                I am specialized in computational epigenomics: the analysis of methylation changes happening throughout
                the
                genome of cancer patients using different bioinformatic and biostatistical methods on data obtained from
                patient tumor samples.</p>
            <p>
                My main expertise is focused on analysis strategies of methylation array (Infinium® Human Methylation
                450K
                BeadChip and Infinium® MethylationEPIC data) and bisulfite sequencing data. More specifically, I am
                interested in
                the methylation changes happening along repetitive elements (also known as "repeats") of the human
                genome in
                cancer patient, in order to improve cancer diagnosis and prognosis.</p>
        </div>
        <div style="float:left; width: 180px;">
            <div style=" margin-left: 10px; margin-right: 10px;">
                <img src="profile_cropped.jpg" style="float: right;" width="160">
            </div>
            <div style="text-align:center;">
                <button
                    style="background-color: steelblue; border-radius: 7px; padding: 6px; border-style: solid; border-color: lightgrey;"><a
                        href="https://cloud.saturday-ns.net/index.php/apps/appointments/pub/HvMCKcWBcsOVV40%2B/form"
                        style="color: white;"><span><strong>Appointment request</strong></span></a></button>
            </div>
        </div>
        <div style="clear: both;"></div>
    </div>
    <p>In order to improve existing methylation analysis strategies, and to study repeats' methylation in
        cancer, I
        also have a developper activity: I design bioinformatic tools to analyze methylation data, and produce
        neat
        visualisations of the data I work on.<br>
        I do so using 2 programming languages: R and Python3.<br>
        All the tools I design are open source, so anyone can find them available online on my <a
            href="https://github.com/YoannPa/Computational_Epigenomics"><strong>Github profile</strong></a> and use it
        at its own
        convenience.</p>
    <strong>Selected publications</strong>
    <p>
        <a href="https://doi.org/10.1093/bioinformatics/bty007">
            Pageaud Y. et al. (2018)
            <em>
                <strong>Enrichment Analysis with EpiAnnotator.</strong>
                Bioinformatics.
            </em>
        </a>
    </p>
    <p><a href="https://academic.oup.com/narcancer/article/3/1/zcab007/6168271?login=true">
            Ibing S. et al. (2021)
            <em>
                <strong>On the impact of batch effect correction in TCGA isomiR expression data.</strong>
                NAR Cancer.
            </em>
        </a>
    </p>
    <p><a href="https://zenodo.org/record/5010925">Breuer K. et al. (2021)
            <em>
                <strong>workflUX - An open-source, cloud-ready web application for simplified deployment of big data
                    workflows.</strong>
                Zenodo.
            </em>
        </a>
    </p>

    <strong>Selected tools I developped</strong>
    <p><em><a href="https://github.com/YoannPa/BiocompR"><strong>BiocompR</strong></a></em> is an R package built upon
        ggplot2, and using data.table. It improves some visualisations commonly used in biology and genomics for data
        comparison and dataset exploration, introduces new kind of plots, provides a toolbox of functions to work with
        ggplot2 and grid objects, and ultimately, allows users to customize plots produced into publication ready
        figures.</em></p>
    <p><em><a href="https://github.com/YoannPa/methview.qc"><strong>Methview.qc</strong></a></em> allows you to run
        quality
        control analysis on your methylation array dataset, and to collect all results in neat ready-to-publish
        plots.</p>

    <p><em><a
                href="https://academic.oup.com/bioinformatics/article/34/10/1781/4796958?login=false"><strong>EpiAnnotator</strong></a></em>
        is an R Package accompanied by a web interface. It contains regularly updated annotations from 4 public
        databases: Blueprint, RoadMap, GENCODE and the UCSC Genome Browser. Annotations are hosted locally or in a
        server environment and automatically updated by scripts of our own design. Thousands of tracks are
        available, reflecting data on a variety of tissues, cell types and cell lines from the human and mouse
        genomes. Users need to upload sets of selected and background regions. Results are displayed in customizable
        and easily interpretable figures.<br>
        <a href="http://epigenomics.dkfz.de/EpiAnnotator/">EpiAnnotator DKFZ web app available here!</a>
    </p>
    <p><em><a href="https://github.com/YoannPa/NCBI.BLAST2DT"><strong>NCBI.BLAST2DT</strong></a></em> is an R package
        allowing you to submit DNA sequences to NCBI BLAST servers directly from the console, to retrieve potential
        hits on a genome or sequence database, and to collect all results within an R data.table.<br>
        It makes use of the R package hoardeR to submit sequences to the NCBI BLAST API, and then parses the XML
        BLAST results returned to load them as an R data.table to make it more easy to query, sort, order and subset
        the resulting hits.
    </p>
    <p><em><a href="https://github.com/YoannPa/biotab.manager"><strong>Biotab.manager</strong></a></em> is an R package
        allowing you to download, manage, subset, and aggregate TCGA patients clinical data (biotabs) from the GDC
        portal. The package is built upon TCGAbiolinks to query TCGA databases, and makes use of R data.table handle
        queries results.</p>
    <strong>Education</strong>
    <p><strong>2016 - 2017</strong>: 2nd year of MSc in Structural Bioinformatics at Université Paris Diderot (Paris 7),
        France.
    </p>
    <p><strong>2015 - 2016</strong>: 1st year of MSc in Genomics at Université Paris Saclay, France.
    </p>
    <p><strong>2014 - 2015</strong>: 3rd year of BSc: Genetic and Bioinformatics at Genopole, France.
    </p>
    <p><strong>2012 - 2014</strong>: 1st & 2nd years of BSc: Fundamental & Biomedical Sciences at Université Paris
        Descartes (Paris 5), France.</p>
    <strong>Follow me on</strong>
    <ul>
        <li><a href="https://www.linkedin.com/in/yoannpageaud/">LinkedIn</a></li>
        <li><a href="https://github.com/YoannPa">GitHub</a></li>
        <li><a href="https://genomic.social/@YoannPageaud">Mastodon</a></li>
        <li><a href="https://www.researchgate.net/profile/Yoann-Pageaud">ResearchGate</a></li>
    </ul>

</body>

</html>